src/libsystemd-network/test-ndisc-ra.c

```diff
@@ -106,6 +106,11 @@ static struct {
           false },
 };
 
+static struct in6_addr test_rdnss = { { { 0x20, 0x01, 0x0d, 0xb8,
```

Comment:
make this const maybe? that way the missing const on the function would trip up, see other comment ;-)

Reply:
I'd still make this const btw...

Reply:
Yes, finally...